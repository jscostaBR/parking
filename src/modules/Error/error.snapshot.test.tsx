import * as React from 'react'
import { Error } from '.'
import renderer from 'react-test-renderer'

describe('render Error', () => {
  it('render correctly', () => {
    const tree = renderer.create(<Error errorMessage='error'/>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
