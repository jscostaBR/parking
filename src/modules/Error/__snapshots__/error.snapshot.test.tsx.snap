// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Error render correctly 1`] = `
<div
  className="sc-bczRLJ jHnmht"
>
  <img />
  <span>
    error
  </span>
</div>
`;
